### PR TITLE
Use /usr/bin/fdb by default for FDB_NETWORK_OPTION_EXTERNAL_CLIENT_DIRECTORY

### PR DESCRIPTION
**Pull request by ltsampros**
Use /usr/bin/fdb by default for FDB_NETWORK_OPTION_EXTERNAL_CLIENT_DIRECTORY

# Description
fixes: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/807

## Type of change
- Other

# Testing
Local manual testing of the config/samples

### CODE REVIEW

**Review by brownleej — APPROVED** [review]
👍

File: setup/setup.go [hunk 1]
```diff
@@ -249,6 +264,8 @@ func moveFDBBinaries() error {
 			if err != nil && !os.IsNotExist(err) {
 				return err
 			}
+			defer versionBinFile.Close()
```
[the remark this answers is not in the document]
Comment (ltsampros) [reply setup/setup.go:267]: There is no reason to defter closing the files, so I just close the relevant files after we stop needing them. I'd prefer to refactor this function on a separate PR.

[the remark this answers is not in the document]
Comment (johscheuer) [reply setup/setup.go:267]: The `defer` is still here? Did you forget to push your changes?

[the remark this answers is not in the document]
Comment (ltsampros) [reply setup/setup.go]: Pushed :)

**Review by johscheuer — APPROVED** [review]
Changes look fine. Could you take a look why the e2e pipeline failed?

**Pull request merged**